AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 ||boschwiperblades.com/assets/cookie-popup/bosch-privacy-settings-*.min.js
+/whCookieManager$domain=donbosco-medien.de|wirth-horn.de
```

Comment:
I think we should add to cookie_general:
`||cookiemanager.wirth-horn.de^$third-party`

and please add rules for share buttons and newsletter popup

Reply:
@wout-codes 
What about share buttons and newsletter popup?

Reply:
I can't reproduce.